wordpress-mobile/WordPress-iOS
# Show unified Google view

Ref #https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/282
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/289

This adds a `FeatureFlag` to enable the unified Google flow, and updates Auth to display the new view accordingly.

**NOTE:** The Feature Flag is enabled for `localDeveloper` for testing purposes. However, because there is no functionality yet, I will change that to `false` before merging as to not break Google auth for devs.

To test:

---
As noted on the Auth PR, the view doesn't do any Google authentication yet. So just verify that any entry into Google shows this view. Specifically:
- Log In > Continue with Google
- Sign up > Continue with Google

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-28 at 15 14 47](https://user-images.githubusercontent.com/1816888/83194771-6bd8f100-a0f6-11ea-8f14-bfaaa19a3bf8.png)

---
Disable the feature flag. Verify login and signup with Google show the old views. That is:

| Login | Signup |
|--------|-------|
| ![login](https://user-images.githubusercontent.com/1816888/83195089-e86bcf80-a0f6-11ea-964b-1ff72f6d9a26.png) | ![signup](https://user-images.githubusercontent.com/1816888/83195107-ee61b080-a0f6-11ea-9115-2c90907c6d20.png) |

---
PR submission checklist:
